### PR TITLE
[3.3.3] updated Californium and Leshan versions

### PR DESCRIPTION
**Pull request by YevhenBondarenko**

### CODE REVIEW

File: application/src/test/java/org/thingsboard/server/transport/lwm2m/client/LwM2mBinaryAppDataContainer.java [hunk 1]
```diff
@@ -81,8 +81,11 @@ public LwM2mBinaryAppDataContainer() {
     public LwM2mBinaryAppDataContainer(ScheduledExecutorService executorService, Integer id) {
         try {
             if (id != null) this.setId(id);
-            executorService.scheduleWithFixedDelay(() ->
-                    fireResourcesChange(0, 2), 1800000, 1800000, TimeUnit.MILLISECONDS); // 30 MIN
+            executorService.scheduleWithFixedDelay(() -> {
+                        fireResourceChange(0);
+                        fireResourceChange(2);
+                    }
+                    , 1800000, 1800000, TimeUnit.MILLISECONDS); // 30 MIN
```
Comment (nickAS21) [inline application/src/test/java/org/thingsboard/server/transport/lwm2m/client/LwM2mBinaryAppDataContainer.java:88]: fireResourcesChange(getResourcePath(0), getResourcePath(2)), 1800000, 1800000, TimeUnit.MILLISECONDS)

Comment (YevhenBondarenko) [reply application/src/test/java/org/thingsboard/server/transport/lwm2m/client/LwM2mBinaryAppDataContainer.java:88]: it`s the same

**Comment by YevhenBondarenko** [comment]
#5868

File: application/src/test/java/org/thingsboard/server/transport/lwm2m/client/LwM2mBinaryAppDataContainer.java (hunk 1 above)
Comment (boaks) [reply application/src/test/java/org/thingsboard/server/transport/lwm2m/client/LwM2mBinaryAppDataContainer.java:88]: `... , 30, 30, TimeUnit.MINUTES)`

**Pull request merged**